package.json

```diff
@@ -49,7 +49,7 @@
     "raw-loader": "^0.5.1",
     "share": "0.7.27",
     "style-loader": "^0.8.3",
-    "treebeard": "git://github.com/caneruguz/treebeard.git#7042be173a813bd7072e8345f96e7dbbb7fac551",
+    "treebeard": "git://github.com/caneruguz/treebeard.git#01707a0824f48c48d536d238ad3a0f92cc936f5c",
```

Comment:
Does this need to be updated again? I see that there was a commit after this one (https://github.com/caneruguz/treebeard/commit/8f984ef1de7d148c42d61f1156a5e076455aedfb).